scripts/buildSitemap.js

```diff
@@ -1,10 +1,11 @@
 const fs = require('fs');
 const axios = require('axios');
 const path = require('path');
+const tunnel = require('tunnel');
 
 const pages = require('./pages');
 
-const siteUrl = 'https://www.usaspending.gov';
+const siteUrl = 'https://www.usaspending.gov:443';
```

Comment:
This should not be changed. This is the url which is inserted into the `.xml` file iteslf. see ~line 168